def error_message():
    print("Something went wrong")


def addition(first_number, second_number):
    answer = first_number + second_number
    return answer


#error_message()

additionTotal = addition(3, 9)

print("Number to add is: ", additionTotal)
